mantidproject/mantid
# Re #20071: ISIS Reflectometry: Allow autoreduction to pause reduction

Requires #20255

Modifies the Reflectometry GUI such that the autoreduction button toggles to, and can be used as a pause button while processing. 

**To test:**
  - Login into the ICAT catalog.
  - The interface can be accessed from Interfaces->Reflectometry->ISIS Reflectometry.
  - Select `POLREF`, enter investigation id `1520249` and click on the `autoreduce` button to begin autoreduction.
  - Ensure that the `autoreduce` button has changed into a pause button.
  - Click the (autoreduce/pause) button again and ensure that processing resumes.
  - Wait for the processing to complete and ensure that the autoreduce/pause button shows changes to an 'autoreduce' state once again.



Resolves #20071. 

**Release Notes** 

Does not need to be in the release notes?

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
